Makefile

```diff
@@ -75,9 +76,13 @@ SHADER_OBJS = \
 	$(SRCDIR)/shaders/textured_pixelshader.inl \
 	$(SRCDIR)/shaders/untextured_pixelshader.inl
 
-DEBUG := y
```

Comment:
Possibly a controversial change. My preference is to have release builds as the default, so I've removed the definition here.

Reply:
I think it's reasonable to flip now that this tester is being used for something other than supporting my personal dev efforts.